charts/atlantis/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: v0.29.0
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 5.5.1
+version: 5.6.1
```

Comment:
It's common to reset patch when minor is increased: 🙂
```suggestion
version: 5.6.0
```

Reply:
That's what I get for rushing

Reply:
Alll Set! squashed my commits too.